docs/reference/modules/cross-cluster-search.asciidoc

```diff
@@ -125,3 +125,8 @@ will be prefixed with their remote cluster name:
   `node.attr.gateway: true` such that only nodes with this attribute will be
   connected to if `search.remote.node.attr` is set to `gateway`.
 
+`search.remote.connect`::
+  If set to `false` the node will not connect to any configured remote cluster. This is useful to disable cross cluster
```

Comment:
How about:

By default, any node in the cluster can act as a cross-cluster client and connect to remote clusters.  The `search.remote.connect` setting can be set to `false` (defaults to `true`) to prevent certain nodes from connecting to remote clusters.  Cross-cluster search requests must be sent to a node that is allowed to act as a cross-cluster client.